library/azure_rm_securitygroup.py

```diff
@@ -373,6 +388,11 @@ def check_plural(src, dest):
     check_plural('source_port_range', 'source_port_ranges')
     check_plural('destination_port_range', 'destination_port_ranges')
 
+    # when source(destination)_application_security_groups set, remove the default value * of source(destination)_address_prefix
```

Comment:
is it needed?

Reply:
yes, because source_application_security_group is mutual exclusive with source_address_prefix, source_address_prefixes